scripts/gen-edpm-baremetal-kustomize.sh

```diff
@@ -50,7 +44,7 @@ resources:
 namespace: ${NAMESPACE}
 patches:
 - target:
-    kind: ${KIND}
+    kind: OpenStackDataPlaneNodeSet
```

Comment:
Though we can hardcode this, it's set in https://github.com/openstack-k8s-operators/install_yamls/blob/main/Makefile#L573

Reply:
I have set this back to ${KIND}